src/Dropdown.js

```diff
@@ -0,0 +1,177 @@
+import { useState } from "react";
+
+const Dropdown = ({getBreed}) => {
+    const [userChoice, setUserChoice] = useState('placeholder')
+    const [nameSelection, setNameSelection] = useState('')
+
+    const handleUserChoice = (e) => {
+        setUserChoice(e.target.value)
+        setNameSelection(e.target.selectedOptions[0].label)
+    }
+    const handleSubmit = (e) => {
+        getBreed(e, userChoice, nameSelection)
+        setUserChoice('placeholder')
+    } 
+    
+    return(
+        <div className="formWrapper">
+        <form onSubmit={ handleSubmit }>
+            {/* <h2>Show me some cute doggies:</h2> */}
+            <select  id="dogBreed" name="dogBreed" onChange={handleUserChoice} value={userChoice} >
```

Comment:
Adding in some aria attributes to the form or options would be great for accessibility! The contrast of the color palette (blue/yellow) is great.